components/AlphabetGlossary.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+    <div>
+      <div class="text-right mb-5" :style="{marginRight: '10%' }">
+        <button @click="expandAll" class="hover:text-gray-400 text-gray-800 font-bold py-2 px-1 underline text-xs">Expand All</button>
+        <button @click="collapseAll" class="hover:text-gray-400 text-gray-800 font-bold py-2 px-1 underline text-xs">Collapse All</button>
+      </div>
+      <div :style="{ marginLeft: '10%', marginRight: '10%' }">
+        <div v-for="letterObj in letters" :key="letterObj.letter">
+            <h2 @click="toggleShowingLetter(letterObj.letter)" class="flex justify-between items-center cursor-pointer px-4">
+                <span>{{ letterObj.letter }}</span>
+                <span v-if="letterObj.expanded">-</span><span v-else>+</span>
+            </h2>
+          <hr class="mb-3">
+          <ul v-show="letterObj.expanded" class="mb-3">
+            <li v-for="item in sortedGlossary[letterObj.letter]" :key="item.word" class="mb-3">
+              <h3 class="font-semibold">{{ item.word }}</h3>
+              <p><span class="font-medium">Category:</span> {{ item.category }}</p>
+              <p><span class="font-medium">Definition:</span> {{ item.definition }}</p>
+            </li>
+          </ul>
```

Comment:
Just selected a random line to flag this but I can't get this to run. I'm getting the following error:

``
The requested module '@nuxt/kit' does not provide an export named 'addServerMiddleware'                                                                                    09:19:11

  import { defineNuxtModule, resolvePath, resolveAlias, createResolver, requireModule, addTemplate, isNuxt2, installModule, addServerMiddleware } from '@nuxt/kit';
  ^^^^^^^^^^^^^^^^^^^
  SyntaxError: The requested module '@nuxt/kit' does not provide an export named 'addServerMiddleware'
  at ModuleJob._instantiate (node:internal/modules/esm/module_job:134:21)
  at async ModuleJob.run (node:internal/modules/esm/module_job:217:5)
  at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
  at async ModuleContainer.addModule (node_modules/@nuxt/core/dist/core.js:170:20)
  at async ModuleContainer.ready (node_modules/@nuxt/core/dist/core.js:34:7)
  at async Nuxt._init (node_modules/@nuxt/core/dist/core.js:347:5)
```

Was wondering if you have any ideas? Thanks Ruby